tests/BrowserSelenium/BaseTestCase.php

```diff
@@ -24,7 +25,10 @@ public function setUp()
         // Set base url according to environment
         switch (ConfigProvider::getInstance()->env) {
             case 'dev':
-                self::$baseUrl = env('APP_URL');
+                self::$baseUrl = 'http://monica.test/'; // env('APP_URL');
```

Comment:
@asbiin this will be a problem. Shouldn't we put `env('APP_URL')` back? Or actually, even better: `config('app.url')`?

Reply:
Yes you're right !
Actually, it didn't work with `env('APP_URL')`, but I will try with `config('app.url')`.
It seems the laravel framework is not running when the test is running, so I'm not sure how to get the config.